w00_init/t01_big_picture.tex

```diff
@@ -247,7 +247,7 @@
 		\end{itemize}
 		\item Basics of \alert{Machine Learning} (mandatory)
 		\begin{itemize}
-			\item Classification, regression, clustering, decision tree, training-test split, cross validation, pre-processing \ldots
+			\item Classification, regression, clustering, decision tree, training-test split, cross validation, preprocessing \ldots
```

Comment:
I would say both works, I don't think this is a mistake (two other instances of this below I haven't commented on).